internal/config.go

```diff
@@ -52,6 +52,7 @@ const (
 	PgDatabaseSetting            = "PGDATABASE"
 	PgSslModeSetting             = "PGSSLMODE"
 	PgSlotName                   = "WALG_SLOTNAME"
+	PgWalSize                    = "WALG_PG_WAL_SIZE"
```

Comment:
The actual value MUST be taken from pg_control.

Reply:
We cannot read pg_control during wal-push and wal-fetch.
But we can check config agains pg_control during backup-push.

Reply:
99% of users won't need to change this value. One change could be changing the error message `bytes_per_wal_segment of the server does not match expected value` to indicate that a user might need to set this value, which will happen on backup-push if the segment size doesn't match.

Reply:
Oh, I did not realize that this check is already in place..

Reply:
Yes, rewording this would be good. Before this patch user could not do anything, that's why I written it this way...

Reply:
Committed this change + docs